### ISSUE TITLE
Get Log Data of Root Beer

### ISSUE THREAD

**Issue by nauhalf** [scottyab/rootbeer#166]
Is there any method / property to get the logs (String / List<String) from Root Beer when the lib detected root ? 

I want to store the log to firebase so If there was a false positive I can give the information in issue. 

Thx

**Comment by scottyab** [comment]
Good idea. There isn't anything at the moment. Partly why we allow added/allow the individual root checks to be called is that callers could build up a matrix of which checks passed/failed and use that as a guide to which checks were providing false positives. But yes being able to get a String of the checks performed and the output could be useful. Thanks for the suggestion.

**Comment by nauhalf** [comment]
@scottyab I look into the RootBeer Class that you call `QLog.v(completePath + " binary detected!");` to log when rootbeer detected something. Can I just add another property like List<String> logs to put that log into the list and add another function to get the list? if that's possible, I want to make a pull request about it

**Comment by test-pad** [comment]
Hi.
May I know if there is a log file or some means for me to know what is causing the false positive for my Samsung note 20 ultra phone? It is not rooted and trying to find what causes the false positive as some apps cannot run (instarem) or some functions cannot be used properly (shopee top-up).

Thank you very much.

![Screenshot_20210916-065916_RootBeer Sample.jpg](https://user-images.githubusercontent.com/50765030/133523894-c4391695-e508-470b-9017-56a23ae57ac8.jpg)
